tutorials/getstarted.qmd

```diff
@@ -1,3 +1,312 @@
 # 🚀 Get Started with LieGroups.jl
 
-In this tutorial, we aim to present how to get started with `LieGroups.jl`.
\ No newline at end of file
+## Introduction
+
+This tutorial introduces both a few basics of [Lie groups](https://en.wikipedia.org/wiki/Lie_group) as well as how to use [LieGroups.jl](https://juliamanifolds.github.io/LieGroups.jl/stable/). The Lie groups we consider are the rotations in the plane,
+or [special orthogonal group](../groups/special_orthogonal_group.md) as well as rigid body motions,
+or in other words the [special euclidean group](../groups/special_euclidean_group.md).
+
+In a nutshell a Lie group $𝒢$ is a [manifold](https://en.wikipedia.org/wiki/Manifold) $ℳ$ that also is a group a group operation.
+A manifold informally be described as a set that “locally looks like a Euclidean space”
+and has some way to measure angles at every point, to be precise an inner product, that is allowed to (smoothly)
+depend on the point we are at.
+A [group](https://en.wikipedia.org/wiki/Group_(mathematics)) means that on the set there also
+exists an operation, within this package usually denoted by $∘: 𝒢 × 𝒢 → 𝒢$$,
+`LieGroups.jl` uses the manifolds defined in [`Manifolds.jl`](https://juliamanifolds.github.io/Manifolds.jl/stable/). For more details on that, see also the [introductory tutorial](https://juliamanifolds.github.io/Manifolds.jl/stable/tutorials/getstarted/) there.
+A Lie group is usually written as a tuple $(ℳ,∘)$ os the manifold and the group operation thereon.
+
+For more theoretical background, see for example [HilgertNeeb:2012](@cite), especially their Chapter 9.
+
+## Rotations on the plane $ℝ^2$ and in $ℝ^3$.
+
+This first part considers rotations in the plane. These can be represented by rotation matrices
+```math
+R_α = \begin{pmatrix} \cos(α),& -\sin(α)\\ \sin(α) & \cos(α) \end{pmatrix},\qquad α ∈ ℝ,
+```
+
+where we already represent all possible rotations when restricting $α$ to $[0,2π]$.
+Adding two matrices $R_α + R_β$ does not yield a valid rotation matrix again, but we
+can have these matrices in mind as being “locally like the real line”, since changing the rotation angle
+slightly yields rotation matrices that are “close by” in the sense that they perform nearly the same rotation
+when applied to a point $x ∈ ℝ^2$ by computing $R_αx$.
+Matrix multiplication $R_β = R_{α_1}*R_{α_1}$ does yield a new rotation and also fulfils all other
+properties to yield a group operation. Using trigonometric identities we observe that $β = α_1 + α_2$.
```

Comment:
```suggestion
properties to yield a group operation. Using trigonometric identities we observe that $β = α_1 + α_2 \pmod{2\pi}$.
```

Reply:
Feel free to phrase it differently but I think it's important to make it more precise. We can either assume that $\beta=\alpha_1 + \alpha_2$ and conclude that the matrices multiply accordingly, or start from matrix multiplication (like you did) and conclude that we can only establish equality up to a multiple of $2\pi$.

Reply:
I will think about it and agree, it should be more precise and clear.